apps/codeforafrica/content/projects/africanDrone.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
Shouldn't the filename be lower case i.e. `africandrone`?

Reply:
Netlify uses `"identifier_field": "name",` (can be changed to another field) to determine the filename hence the name since the project name is `africanDrone`

Reply:
No, I'm cool with it using name @esirK ... I'm just surprised that `africanDRONE` would result in `africanDrone` while `WanaData` would result in `wanadata`. I was under the impression the "slugify" function that NetlifyCMS uses would lower-case the names first.

Reply:
lemme check that. Might be the file was created before I specified the `identifier_field`